hub-backend/src/main/java/org/panda_lang/hub/HubConfiguration.kt

```diff
@@ -58,42 +58,41 @@ internal class HubConfiguration : WebMvcConfigurer {
 
     @Bean
     fun messageSource(): MessageSource {
-        val messageSource = ReloadableResourceBundleMessageSource()
-        messageSource.setBasename("classpath:messages")
-        messageSource.setDefaultEncoding("UTF-8")
-        return messageSource
+        return ReloadableResourceBundleMessageSource().also {
+            it.setBasename("classpath:message")
+            it.setDefaultEncoding("UTF-8")
+        }
```

Comment:
Kotlin has a set of Scope Functions which are neat and especially useful for initialization or modification on newly created objects which makes the code easier to grasp.

https://kotlinlang.org/docs/reference/scope-functions.html